src/sinks/kafka/service.rs

```diff
@@ -62,15 +62,13 @@ impl Finalizable for KafkaRequest {
 #[derive(Clone)]
 pub struct KafkaService {
     kafka_producer: FutureProducer<KafkaStatisticsContext>,
-    bytes_sent: Registered<BytesSent>,
 }
 
 impl KafkaService {
-    pub(crate) fn new(kafka_producer: FutureProducer<KafkaStatisticsContext>) -> KafkaService {
-        KafkaService {
-            kafka_producer,
-            bytes_sent: register!(BytesSent::from(Protocol("kafka".into()))),
```

Comment:
Sorry if I'm forgetting this from another PR, but where did this get moved to?

Reply:
`BytesSent` is emitted in the driver framework if the trait function is implemented. So by implementing `DriverResponse::bytes_sent()` (above at line 51) , the driver frame work emits it there.